edxval/tests/test_api.py

```diff
@@ -1948,7 +1948,7 @@ def test_get_available_transcript_languages(self):
         # `non_existent_video_id` that does not have transcript
         video_ids = ['super-soaker', self.video_id, dupe_lang_video_id, 'non_existent_video_id']
         transcript_languages = api.get_available_transcript_languages(video_ids=video_ids)
-        self.assertItemsEqual(transcript_languages, ['de', 'en', 'ur'])
+        self.assertEqual(sorted(transcript_languages), sorted(['de', 'en', 'ur']))
```

Comment:
Any particular reason for `self.assertEqual` instead of `self.assertItemsEqual`?